deno/get_price/src/mod.ts

```diff
@@ -0,0 +1,77 @@
+export default async function (req: any, res: any) {
+  const CoinAPIkey = req.env["COIN_API_KEY"];
+  const AlphavantageAPIkey = req.env["ALPHAVANTAGE_API_KEY"];
+  const GOLDAPIkey = req.env["GOLD_API_KEY"];
+
+  const { type } = req.payload;
+
+  if (!type) {
+    throw new Error("type is required");
+  }
+
+  if (!CoinAPIkey || CoinAPIkey === "") {
+    throw new Error("CoinAPI key is required");
+  }
+
+  if (!AlphavantageAPIkey || AlphavantageAPIkey === "") {
+    throw new Error("AlphavantageAPI key is required");
+  }
+
+  if (!GOLDAPIkey || GOLDAPIkey === "") {
+    throw new Error("GOLDAPIkey key is required");
+  }
+
+  let price;
+  if (type.toLowerCase() === "google" || type.toLowerCase() === "amazon") {
+    let code = type.toLowerCase() === "google" ? "GOOGL" : "AMZN";
+    const str = "Time Series (5min)";
+    let response: { [str: string]: any } = await fetch(
+      `https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=${code}&interval=5min&apikey=${AlphavantageAPIkey}`,
+      {
+        method: "GET",
+        headers: {
+          "User-Agent": "request",
+        },
+      },
+    );
+    const data = await response.json();
```

Comment:
Please add some error handling so you can return a properly formatted function response. Same for the other API calls